src/main/java/org/jboss/windup/web/keycloaktool/commands/AddUser.java

```diff
@@ -59,14 +59,16 @@ public static void createUser(File addUserFile, String realmName, String userNam
         user.setUsername(userName);
         user.setCredentials(new LinkedList<CredentialRepresentation>());
 
-        UserCredentialValueModel credentialValueModel = new Pbkdf2PasswordHashProvider().encode(password, iterations > 0 ? iterations : DEFAULT_HASH_ITERATIONS);
+        //UserCredentialValueModel credentialValueModel = new Pbkdf2PasswordHashProvider().encode(password, iterations > 0 ? iterations : DEFAULT_HASH_ITERATIONS);
 
         CredentialRepresentation credentials = new CredentialRepresentation();
-        credentials.setType(credentialValueModel.getType());
-        credentials.setAlgorithm(credentialValueModel.getAlgorithm());
-        credentials.setHashIterations(credentialValueModel.getHashIterations());
-        credentials.setSalt(Base64.encodeBytes(credentialValueModel.getSalt()));
-        credentials.setHashedSaltedValue(credentialValueModel.getValue());
+        credentials.setType("password");
+        //credentials.setAlgorithm(credentialValueModel.getAlgorithm());
+        //credentials.setHashIterations(credentialValueModel.getHashIterations());
+        //credentials.setSalt(Base64.encodeBytes(credentialValueModel.getSalt()));
+        //credentials.setHashedSaltedValue(credentialValueModel.getValue());
+        credentials.setValue(password);
+        credentials.setTemporary(false);
 
```

Comment:
So it is now not salted in code, but keycloak handles that by default?

Reply:
Yeah, Keycloak will salt them when it stores it.